docker/app/Dockerfile

```diff
@@ -10,6 +10,9 @@ RUN npm config set unsafe-perm true && npm install -g npm@7.6.3
 RUN mkdir -p /data
 WORKDIR /data
 
+# copy npm cache directory; as optimization for npm install
+COPY --from=megahirt/web-languageforge:npm-cache /root/.npm /root/.npm
```

Comment:
Once I know how to publish to sillsdev, we can change this away from the megahirt repo

Reply:
I think this is promising, I'd like to suggest this be done via a GHA triggered off any push to `develop` and producing a `npm-cache-develop` tag.

Additionally, I'd like to consider/discuss the implications this may have when producing the "production" version of this image.

Reply:
I don't see any issues with using this npm-cache in a production build.  It's a cache that npm should be able to intelligently use at it's discretion, but should not affect the product output or final app image in any way (thanks to multi-stage builds!).  The cache folder (`~/.npm`) will be generated either way.  Saving it as a docker image instead of having to rebuild it on every run should save time in both prod and dev image builds.  That's my thinking anyway.

Reply:
I like the idea of of publishing a GHA triggered npm-cache-develop image after we have proved out the concept.

Reply:
yep, I was not thinking about it properly, thanks for reminding me this is transient ad won't impact our app integrity across versions.